src/app/hierarchical-grid/hierarchical-grid-filtering-template/hierarchical-grid-filtering-template.component.ts

```diff
@@ -34,7 +34,9 @@ export class HGridFilteringTemplateSampleComponent implements OnInit {
         };
     }
 
-    public formatter = (a) => a;
+    public formatter(date: Date): string {
```

Comment:
This change is wrong - this formatter is used in columns like Debut, Launch Date, etc. They are supposed to show the year only. Currently, the formatter doesn't handle well the underlying value and always shows 1/1/1970.